packages/lesmis/src/api/queries/useEntitiesData.js

```diff
@@ -26,7 +26,7 @@ export const useEntitiesData = (entityCode, params = DEFAULT_PARAMS) => {
 };
 
 const PROJECT_PARAMS = {
-  fields: 'id,child_codes,code,country_code,image_url,name,type,parent_code',
+  fields: 'id,child_codes,code,country_code,image_url,name,type,parent_code,attributes',
```

Comment:
I needed to have the attributes to read whether the parent district of a school is a priority district